src/Amazon.ElasticBeanstalk.Tools/Commands/CommandProperties.cs

```diff
@@ -92,6 +93,8 @@ internal void ParseCommandArguments(CommandOptions values)
                 this.LoadBalancerType = tuple.Item2.StringValue;
             if ((tuple = values.FindCommandOption(EBDefinedCommandOptions.ARGUMENT_ENABLE_STICKY_SESSIONS.Switch)) != null)
                 this.EnableStickySessions = tuple.Item2.BoolValue;
+            if ((tuple = values.FindCommandOption(EBDefinedCommandOptions.ARGUMENT_DISABLE_IMDS_V1.Switch)) != null)
```

Comment:
The `PersistSettings` method also needs to be updated for the new switch.
https://github.com/aws/aws-extensions-for-dotnet-cli/blob/master/src/Amazon.ElasticBeanstalk.Tools/Commands/CommandProperties.cs#L103

Reply:
Done in latest commit